lib/active_fedora/file/attributes.rb

```diff
@@ -72,8 +72,8 @@ def default_mime_type
     end
 
     def fetch_mime_type
-      return default_mime_type if new_record?
-      ldp_source.head.content_type
+      return default_mime_type if new_record? && metadata.mime_type.blank?
+      metadata.mime_type.first
```

Comment:
@jcoyne what do you think about not calling `.first` here and just returning the array? All of the other terms in DefaultSchema and additional terms added via HydraWorks are returned as arrays. We can leave it be, but I'd have to make special consideration for it in HydraWorks, since it expects arrays from everything else.